arangod/Graph/Steps/BaseStep.h

```diff
@@ -25,7 +25,7 @@
 
 #include "Basics/ResultT.h"
 
-#include <velocypack/HashedStringRef.h>
+#include "velocypack/HashedStringRef.h"
```

Comment:
```suggestion
#include <velocypack/HashedStringRef.h>
```
Please rollback that change.